Added CHANGELOG.md for v0.1.0 release (#40)
gauravgahlot/argoflows: CHANGELOG.md

# Changelog

All notable changes to this project are documented in this file.

This project follows [Keep a Changelog](https://keepachangelog.com/) conventions and adheres to [Semantic Versioning](https://semver.org/).

> ## [v0.1.0] - 2023-09-20

Reviewer: ```suggestion
## [v0.1.0] - 2025-01-18
```

### Initial Release

> Basic Python client for Argo Workflows with essential API operations.

Reviewer: ```suggestion
Argoflows provides a Rust client library for interacting with Argo Workflows.
```

#### Added Features

> ##### **Workflow Service**

Reviewer: ```suggestion
##### Workflow Service
```

Also, add a new line after the heading.

- `create_workflow` - Submit workflows to Kubernetes
- `get_workflow` - Retrieve workflow details
- `list_workflows` - List workflows in namespace
- `delete_workflow` - Delete workflows

> ##### **Workflow Template Service**

Reviewer: Same as above.

- `create_workflow_template` - Create workflow templates
- `get_workflow_template` - Get template details
- `list_workflow_templates` - List all templates
- `delete_workflow_template` - Delete templates

> ##### **Archived Workflow Service**

Reviewer: We don't support `ArchivedWorkflowService` yet.
